xwiki-release-scripts/maven-release.sh

```diff
@@ -104,7 +104,7 @@ function check_versions() {
     exit -1
   fi
 
-  if [ $do_release ]
+  if [[ $do_release == 'true' ]]
```

Comment:
Is it on purpose that you sometimes use `=` and sometimes `==` for what seems to be the same need ?

Reply:
No indeed, that's a mistake, fixing this

Reply:
Reading https://unix.stackexchange.com/questions/72039/whats-the-difference-between-single-and-double-equal-signs-in-shell-compari it seems both `=` and `==` are equivalent in our case.
But I replaced all comparison for `==` for consistency.